openlcs/libs/encrypt_decrypt.py

```diff
@@ -0,0 +1,24 @@
+from cryptography.fernet import Fernet
+
+# secret key for auth token encrypt and decrypt
+TOKEN_SECRET_KEY = "RiDMWYCf868Zm7BMbGtHdPF0yR425lB9b4sAiAkbYyM="
```

Comment:
according to the doc, anyone with the key can generate/decrypt messages. Thus it'll be great to move this into settings. Better to be generated on the fly(`Fernet.generate_key()`)

Thanks for the approach. LGTM

Reply:
Add TOKEN_SECRET_KEY to django settings.I tried to generated it use **Fernet.generate_key()**, but since django and celery are different process.Django process and celery process will get different token.So i  used a fixed one.